# 📌 Лабораторна робота №4
## Основи нейронних мереж: побудова та навчання перцептрона

**Мета** Ознайомитись з поняттями штучних нейронних мереж, їхньою
архітектурою, принципами навчання, а також навчитися будувати та навчати
одношаровий та багатошаровий перцептрон для задач бінарної класифікації.

<br>

**Виконав:** ст. гр. ОІ-31 | _Мацишин Михайло_

**Прийняв:** _Лящинський Петро Борисович_

## 1| Отримання та підготовка датасету

In [79]:
print(f"\nКількість записів: {len(df)}")


Кількість записів: 103


## 2| Визначення вхідних параметрів

Додаємо класифікацію довжини термінів для всіх записів.

**Довжина термінів:** Обчислюємо кількість слів у кожному записі для колонок Expedia і Booking.

**Класифікація:** Застосовуємо правила (≤3 — короткий, 4-5 — середній, >5 — довгий) до всіх 103 записів.